### DISCUSSION TITLE
Stylus pen only feature

### DISCUSSION THREAD

**Discussion by nicememe123** [pubpub-zz/ppInk#253] [Q&A] [answered]
Was wondering if there would be support for an option to disable drawing with mouse and touch screen and only enable stylus pen input for drawing? A simple implementation to ignore all mouse and touch screen inputs would mean it would save a lot of keystrokes having to toggle the global hotkey on and off, and help with palm rejection as well.

Thanks for making the fork, really enjoying the program.

**Comment by pubpub-zz** [comment, marked as answer]
thanks for your support. happy you are enjoying

ppInk (as gInk) uses strokes from microsoft.ink (standard library from microsoft). They're is no way to select the input device. Also it processes the events received by the window : therefore I'm not able to capture only some events and leave other to be propagated to windows below.
Sorry
